### PR TITLE
User type and TripCard update

### PR DESCRIPTION
**Pull request by newb1er**

### CODE REVIEW

File: src/modules/types/user.ts [hunk 1]
```diff
@@ -1,6 +1,19 @@
+import { TimeString } from "./route";
```
Comment (ericyangchen) [inline src/modules/types/user.ts:1]: no "./route" file uploaded

**Review by ericyangchen — CHANGES_REQUESTED** [review]

File: src/modules/types/user.ts [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+export type UserId = number;
+export type Name = string;
+export type Email = string;
+export type GoogleId = string;
+export type PictureUrl = string;
+
 export type User = {
-  name: string;
-  avatarSrc?: URL;
+  id: UserId;
```
Comment (ericyangchen) [inline src/modules/types/user.ts:10]: The type can just be `string`, `number`, or some built-in type.

Comment (newb1er) [reply src/modules/types/user.ts:10]: Sure thing, but if the domain become much more complex, it's really hard to maintain each basic types.

e.g. If we change the `UserId` from type `number` to `string`, we need to find out every interfaces or types  that contains `UserId` and change it.

I treat these types more like our custom primitive types.

File: src/modules/types/user.ts (hunk 1 above)
Comment (newb1er) [reply src/modules/types/user.ts:1]: new commit pushed

**Pull request merged**